src/main/java/org/osc/manager/ism/entities/ApplianceManagerConnectorEntity.java

```diff
@@ -0,0 +1,185 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.ism.entities;
+
+import static javax.persistence.FetchType.LAZY;
+
+import java.util.List;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Transient;
+
+import org.osc.sdk.manager.element.ApplianceManagerConnectorElement;
+import org.osc.sdk.manager.element.ManagerTypeElement;
+
+@Entity
+public class ApplianceManagerConnectorEntity implements ApplianceManagerConnectorElement {
+
+    private String name;
+
+    @Id
+    @Column(name = "id")
+    private Long id;
+    @Transient
+    private String ipAddress;
+    @Transient
+    private String username;
+    @Transient
+    private String password;
+    @Transient
+    @Column(name = "api_key")
+    private String ApiKey;
+    @Transient
+    private String lastKnownBrokerIpAddress;
+    @Transient
+    private byte[] publicKey = null;
+    @Transient
+    private String clientIpAddress;
+    @Transient
+    private String serviceType;
+
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = LAZY, mappedBy = "parent")
```

Comment:
The other way is if policies are created before domains are created. one can still map MC's with the policies...